lib/std/bytes_differ.zig

```diff
@@ -0,0 +1,96 @@
+const std = @import("std");
+
+/// If `hexdump` is true, this instead serves only as a hexdump printer.
+pub fn BytesDiffer(comptime hexdump: bool) type {
```

Comment:
Putting this structure here allows us to reuse it in debug.zig as well as testing.zig while not exposing it to the public API surface.